backend/editor/api.py

```diff
@@ -409,8 +400,10 @@ async def edit_entry(request: Request, branch: str, taxonomy_name: str, entry: s
     """
     taxonomy = TaxonomyGraph(branch, taxonomy_name)
     incoming_data = await request.json()
-    result = taxonomy.update_nodes("ENTRY", entry, incoming_data)
-    updated_entry = list(result)
+    log.info("start entry update")
+    updated_entry = await taxonomy.update_nodes("ENTRY", entry, incoming_data)
+    log.info("end entry update")
+    log.info(f"return {updated_entry}")
```

Comment:
Can we remove this log?

Reply:
yes totally ! I added it when I was debugging !